tests/clients/test_base.py

```diff
@@ -58,6 +59,28 @@
 
 HTTP_METHODS = ["GET", "POST", "PUT", "DELETE", "PATCH"]
 
+STREAM_RESPONSE = [
+    "id: 0",
+    "event: mock event",
+    "data: {}",
+    " ",
+    "id: 1",
+    "event: mock event",
+    "data: {}" " ",
+]
```

Comment:
Is this supposed to be:

```suggestion
STREAM_RESPONSE = [
    "id: 0",
    "event: mock event",
    "data: {}",
    " ",
    "id: 1",
    "event: mock event",
    "data: {}",
    " ",
]
```

I think what happened here is that you wrote:

```python
    ...
    "id: 1",
    "event: mock event",
    "data: {}"
    " ",
]
```
(missing the comma from the end of the "data" line)

then probably applied black, which joined the last two lines. The reason that it did this is that two string literals next to each other with no operators in between is valid Python, and they get concatenated, i.e.:

```python
>>> a = "onetwo"
>>> b = "one" "two"
>>> c = "one" + "two"
>>> a == b
True
>>> b == c
True
```

Since there's no comma at the end of the "data" line, Black (correctly) interprets the next line as being the continuation of the same statement and joins them together. Happy to discuss tomorrow if it's not clear why this happened.

Reply:
makes sense, thanks!